opcua/common/node.py

```diff
@@ -315,3 +318,9 @@ def call_method(*args, **kwargs):
         from opcua.common import methods
         return methods.call_method(*args, **kwargs)
 
+    def delete_node(self, nodeid, deleterefs=True):
+        deletenode = ua.DeleteNodesItem()
+        deletenode.NodeId = nodeid
+        deletenode.DeleteTargetReferences = deleterefs
+        results = self.server.delete_nodes([deletenode])
+        results[0].check()
```

Comment:
The idea here was that Node is high level and only uses Node objects, then  session or bclient object is available to give more control to user and accept DeleteNodeItem as argument
Node.delete_nodes([nodes])
server.iserver.isession.delete_nodes(params) ( or client.bclient.delete_nodes(params) on client side)
I should really document that somewhere